packages/cosmos/src/util.ts

```diff
@@ -29,3 +29,37 @@ export function uri(strings: TemplateStringsArray, ...values: any[]) {
   }
   return s;
 }
+
+/**
+ * Parses a connection string for Cosmos DB.
+ * @param connString Cosmos DB connection string.
+ */
+export function parseConnectionString(
+  connString: string
+): CosmosDBConnectionStringParams {
+  const res = {} as CosmosDBConnectionStringParams;
+  connString.split(';').forEach(kv => {
```

Comment:
Code looks great, thanks for all the PRs!  Merged two of them, there's a test failing in this one. 

Could lean on `.split` a little more in this function if you'd like:
```js
const [k1, v1, k2, v2] = connString.replaceAll(';', '=').split('=');
const params = { [k1]: v1, [k2]: v2 } as CosmosDBConnectionStringParams;
```

Reply:
I'm looking at it right now, but I can't figure out what's wrong. I can't run the tests in my environment (`npm run test` fails with a Puppetteer error), so I can't debug the actual test. Running the method itself seems fine however, which is puzzling.

As for that PR... I am concerned that if in the future the format of the ConnString changes and more than 2 things are added inside, this will break.

Reply:
Another problem with this is that the value could contain a `=`

Reply:
ok resolving